runtime/compiler/optimizer/J9ValuePropagation.cpp

```diff
@@ -1651,6 +1714,277 @@ J9::ValuePropagation::constrainRecognizedMethod(TR::Node *node)
       }
    }
 
+void
+J9::ValuePropagation::transformFlattenedArrayElementLoad(TR_OpaqueClassBlock *arrayClass, TR::Node *callNode)
+   {
+   /*
+    * primitive public class MultipleFieldsPrimitive {
+    *    public final int x;
+    *    public final int y;
+    * }
+    * primitive public class NestedFieldsPrimitive {
+    *    public final MultipleFieldsPrimitive vtFiled0;
```

Comment:
Do you mean `vtField0` and `vtField1` (and the 14 other places below that reference these)?

If so, please correct.  These are just comment changes so we don't have to re-run CI.

Reply:
Sorry my bad. The typos are correct in 9675b80.